src/libs/import.c

```diff
@@ -1440,47 +1438,76 @@ static void _update_folders_list(dt_lib_module_t* self)
     _expand_folder(folder, TRUE, self);
   else
     _expand_folder(last_place, FALSE, self);
-  g_free(folder);
 }
 
-static void _add_custom_place(const gchar *folder, dt_lib_module_t* self)
+static void _escape_place_name_comma(char *name)
 {
-  dt_lib_import_t *d = (dt_lib_import_t *)self->data;
-  const char *current_folders = dt_conf_get_string_const("ui_last/import_custom_places");
-  GtkTreeIter iter;
+  for(int i = 0; name && i < strlen(name); i++)
+    if(name[i] == ',') name[i] = '\1';
```

Comment:
Are we sure that \1 is not some kind of prefix in multi-bytes chars in UTF encoding?

Reply:
UTF-8 maybe but in the context of multi-bytes chars. Non-ascii characters are  encoded using 2-4 bytes. And byte inside a multi-byte chars may well be 1 I suppose. And here you read one char by one char so crossing multi-bytes encoding.

To avoid any issue why not use a character that cannot be used to create a filename?

Just checked and this is not easy to have cross-platform solutions :( So yes using `','` may be a good option... Let's see if we can find something that won't break current settings.

Reply:
I see, not problem then. I don't know the code point encoding by memory :) Thanks to have checked.